modules/nw-creating-external-project-and-service.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * ingress/getting-traffic-cluster.adoc
+
+[id="nw-creating-external-project-and-service_{context}"]
+= Creating a project and service
+
+If the project and service that you want to expose do not exist, first create
+the project, then the service.
+
+If the project and service already exist, go to the next step:
+*Exposing the service to create a route*.
+
+. Log into {product-title}.
+
+. Create a new project for your service:
++
+----
+$ oc new-project <project_name>
+----
++
+For example:
++
+----
+$ oc new-project external-ip
```

Comment:
Because this module describes common prerequisites for the `nw-exposing-service`, `nw-using-load-balancer-getting-traffic`, `nw-using-service-external-ip`, and `nw-using-nodeport` modules, I suggest the following changes to reduce risk of confusion:

* Rename this module's file, omitting the word "external: `nw-creating-external-project-and-service`→`nw-creating-project-and-service`.
* Use a different project name: `external-ip`→`myproject`.